docs/en/tutorials/getting-started.rst

```diff
@@ -157,12 +157,12 @@ The second block consists of the instantiation of the ORM
 ``Configuration`` object using the Setup helper. It assumes a bunch
 of defaults that you don't have to bother about for now. You can
 read up on the configuration details in the
-`reference chapter on configuration <../reference/configuration.rst>`_.
+`reference chapter on configuration <http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/configuration.html>`_.
```

Comment:
This one should probably be an relative RST internal link target: http://docutils.sourceforge.net/docs/user/rst/quickref.html#internal-hyperlink-targets